### DISCUSSION TITLE
I cant find the vehicle spawner inside extras-addon

### DISCUSSION THREAD

**Discussion by damiankubets** [Deadlineem/Extras-Addon-for-YimMenu#85] [Q&A]
it only shows Tricks/stunts, tokyo drift and fatbedscrip any help?
![imagen_2024-06-16_023347514](https://github.com/Deadlineem/Extras-Addon-for-YimMenu/assets/125930734/c7dce204-e90e-4cc8-8418-cf504df72ba0)

**Comment by Deadlineem** [comment]
Click a players name, its the blue tab 
![Screenshot_126](https://github.com/Deadlineem/Extras-Addon-for-YimMenu/assets/24372625/bd93e096-5611-488a-b222-6c20f6aba958)

**Reply by damiankubets** [reply]
Thank you so much!

**Comment by aiye400** [comment]
Has the vehicle gifting option been taken out? I can't seem to find it, and no blue vehicle options shows up when I click on a player. Am I missing something?

**Reply by Deadlineem** [reply]
> Has the vehicle gifting option been taken out? I can't seem to find it, and no blue vehicle options shows up when I click on a player. Am I missing something?

The vehicle spawner and gifting has been temporarily disabled since it broke during the update to 1.69, you can enable it again if you want by uncommenting it in extras-addon.lua but it is very unstable

**Reply by aiye400** [reply]
ah gotcha thank you. is it detected or just unstable?
